src/Microsoft.CodeQuality.Analyzers/Core/ApiDesignGuidelines/DoNotDeclareStaticMembersOnGenericTypes.cs

```diff
@@ -54,7 +54,8 @@ public override void Initialize(AnalysisContext analysisContext)
                         (methodSymbol.IsAccessorMethod() ||
                          (methodSymbol.MethodKind == MethodKind.UserDefinedOperator &&
                             (methodSymbol.Name == WellKnownMemberNames.EqualityOperatorName ||
-                             methodSymbol.Name == WellKnownMemberNames.InequalityOperatorName))))
+                             methodSymbol.Name == WellKnownMemberNames.InequalityOperatorName)) ||
+                             (methodSymbol.MethodKind == MethodKind.Conversion)))
```

Comment:
Nit: You probably don't need the parentheses around `methodSymbol.MethodKind == MethodKind.Conversion` and also switch the indentation to same as line 55 above.